README.md

```diff
@@ -1 +1,73 @@
-# AwesomeBooks
\ No newline at end of file
+# Awesome books
+
+>"Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
+In this project, modular JavaScript was written using ES6 modules, npm was used as a software packages system and, library Luxon was also used.
+
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- Git
+- npm
+- Luxon
+
+
+## Live Demo (if available)
+
+[Live Demo Link](https://elsonotake.github.io/AwesomeBooks/)
+
+
+## Getting Started
+
+To get a local copy, follow these simple steps.
+
+### Prerequisites
+
+In order to run this project on your computer, please make sure that you have a web browser installed
+
+### Setup
+
+ - Create a new folder;
+ - Copy the index.html to this folder;
+ - Copy the index.css to this folder;
+ - Copy the index.js to this folder;
+ - Create the modules folder in this folder;
+ - Copy the books.js to the modules folder;
+ - Copy the list_books.js to the modules folder;
+ - Copy the local-storage.js to the modules folder;
+ - Copy the luxon.js to the modules folder;
+ - Copy the user_interface.js to the modules folder;
+ - Double click on the index.html;
+
+Once you completed these steps you should see the site in your browser.
+
+
+## Author
+
+👤 **Elson Otake**
+
+- GitHub: [@elsonotake](https://github.com/elsonotake)
+- Twitter: [@elsonotake](https://twitter.com/elsonotake)
+- LinkedIn: [Elson Otake](https://linkedin.com/in/elson-otake-0b5b9138)
+
+
+## 🤝 Contributing
+
+[HeDevedUp](https://github.com/HeDevedUp) Jesse Uzoma
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## Acknowledgments
+
+- Microverse
```

Comment:
- [ ] Please update your Readme, currently, some headings are missing such as  

``` ##  License ``` section with the MIT license at the end of your README file to make it more professional. I would also recommend you add this. 😄  

- [ ] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc

- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.